clippy_lints/src/lib.rs

```diff
@@ -1894,6 +1893,7 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
     });
 
     let avoid_breaking_exported_api = conf.avoid_breaking_exported_api;
+    store.register_late_pass(move || Box::new(approx_const::ApproxConstant::new(msrv)));
```

Comment:
Could you also add the lint name `APPROX_CONSTANT` to the `msrv` documentation to have it show up in the lint documentation?

https://github.com/rust-lang/rust-clippy/blob/a8c2c7b712c273f13258de8b87b233918d8726b9/clippy_lints/src/utils/conf.rs#L135-L138